09/fish.py

```diff
@@ -0,0 +1,30 @@
+class Fish:
+    def __init__(self, type):
+        self.type = type
+    
+    def swim(self):
+        print('{} can swim in water.'.format(self.type))
+
+class Whales(Fish):
+    def swim(self):
+        super().swim()
+        print('But it\'s not really a fish.')
```

Comment:
Když víš, že uvnitř stringu budeš chtít psát znak _nějakých_ uvozovek, tak bych jako ohraničení stringu použil právě ty druhé uvozovky.

V tomhle případě:

```python
print("But it's not really a fish.")
```

... a naopak by to mohlo být takto:

```python
print('A jakmile otevřel dveře, zvolal: "Auvajs!", načež se chytil za bolavou nohu.')
```

... a pokud se tam vyskytují oboje, tak lze použít i trojité uvozovky:

```python
print("""No představ si, že oba znaky, tedy " i ' lze v Pythonu použít pro vytvoření řetězce.""")  # Nebo klidně '''
```